app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -415,7 +424,10 @@ public function removeMember(TeamRemoveMemberRequest $removeMemberRequest, Team
         $owner = $team->getUserAttribute();
 
         // TODO: need to add leave team functionality
-        if ($owner->id === $auth_user->id || $data['id'] === $auth_user->id || $this->authorize('removeTeamUsers', [Team::class, $team])) {
+        if ($owner->id === $auth_user->id
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line